src/analysis/calculus/fderiv.lean

```diff
@@ -70,6 +74,16 @@ open_locale topological_space
 
 noncomputable theory
 local attribute [instance, priority 10] classical.decidable_inhabited classical.prop_decidable
+-- local attribute [instance, priority 10000] continuous_linear_map.add_comm_group
+--   continuous_linear_map.module normed_field.normed_field normed_field.nondiscrete_normed_field
+-- local attribute [instance, priority 5000] normed_space.to_module
+--   module.to_semimodule semimodule.to_distrib_mul_action distrib_mul_action.to_mul_action
+--   mul_action.to_has_scalar add_comm_group.to_add_comm_monoid
+--   add_comm_monoid.to_add_monoid ring.to_monoid ring.to_semiring
+--   nondiscrete_normed_field.to_normed_field
+--   normed_group.to_add_comm_group normed_field.to_discrete_field discrete_field.to_field
+--   field.to_division_ring field.to_comm_ring comm_ring.to_ring uniform_space.to_topological_space
+--   metric_space.to_uniform_space' normed_group.to_has_norm
```

Comment:
What is all this? If some of them really speed things up, you can keep them, but it is probably worth adding an explanation. Otherwise, please remove.

Reply:
I was trying to fix a failure to find an instance, and forgot to remove an unsuccessful attempt. See `letI := ...` for the correct version.